internal/connor/connor.go

```diff
@@ -8,6 +8,10 @@
 // such as JSON can be filtered as expected.
 package connor
 
+import (
+	"github.com/sourcenetwork/defradb/client/request"
```

Comment:
todo: connor should not depend on defra

Reply:
I instinctively wanted to agree and give this comment a thumbs up, but then I wondered why it actually needs to be so now.

Why do you think it should not be dependent on Defra?

Reply:
AFAIK it was originally third-party dependency.

And there was a time when we played around with transitioning to Rust and we [started with connor package ](https://github.com/sourcenetwork/defradb/pull/2555). This dependency it trivial and should no cause any trouble, as it's just a constant. But other devs might start adding more stuff to it.

Reply:
I removed the dependency by copying the alias constant.